Objects/Converters/ConverterRevit/ConverterRevitShared/Partial Classes/ConvertMaterialQuantities.cs

```diff
@@ -25,16 +25,18 @@ public Objects.Other.MaterialQuantity MaterialQuantityToSpeckle(DB.Element eleme
       //Get quantities
       double volume = element.GetMaterialVolume(material.Id);
       double area = element.GetMaterialArea(material.Id, false); //To-Do: Do we need Paint-Materials
-
+                                                                 //length
+      double length = 0;
+      if (LocationToSpeckle(element) is ICurve) length = (LocationToSpeckle(element) as ICurve).length/1000;  //LocationCurve default is mm (Speckle)
```

Comment:
Not sure about this line. I'll assign @connorivy to give this a quick check before going in.